src/bundle/Resources/views/locationview.html.twig

```diff
@@ -30,7 +30,7 @@
                 {% endfor %}
             </div>
             <div class="ez-tabs-panel" id="ez-tab-details">
-                <p>Some details should be generated here</p>
+                {{ render(controller('ez_content:viewContent', {'contentId': contentId, 'viewType': 'details_tab'})) }}
```

Comment:
`ez_content:viewContent` is deprecated, you must use `ez_content:viewAction`.
Also, you should pass the `locationId` as an extra argument, in order to know which location is being viewed.